packages/core/src/external/surescripts/client.ts

```diff
@@ -318,14 +320,41 @@ export class SurescriptsSftpClient extends SftpClient {
    * @param requester the requester data for Surescripts data
    * @throws an error if the requester's NPI is invalid
    */
-  private validateRequester(requester: SurescriptsRequesterData): void {
+  private async validateRequester(requester: SurescriptsRequesterData): Promise<void> {
     if (!validateNPI(requester.facility.npi)) {
       this.log(`Invalid NPI "${requester.facility.npi}" for CX ID "${requester.cxId}"`);
       throw new MetriportError("Invalid NPI", undefined, {
         npiNumber: requester.facility.npi,
         cxId: requester.cxId,
       });
     }
+    const isSurescriptsEnabled = await isSurescriptsFeatureFlagEnabledForCx(requester.cxId);
+    if (!isSurescriptsEnabled) {
+      this.log(`Surescripts is not enabled for cx "${requester.cxId}"`);
+      throw new MetriportError("Surescripts is not enabled", undefined, {
+        cxId: requester.cxId,
+      });
+    }
+    if (!this.isSurescriptsAllowedForOrgType(requester.org.type)) {
+      this.log(
+        `Cannot make Surescripts requests for cx "${requester.cxId}" with org type "${requester.org.type}"`
+      );
+      throw new MetriportError("Invalid Surescripts organization type", undefined, {
+        cxId: requester.cxId,
+        orgType: requester.org.type,
+      });
+    }
+  }
+
+  private isSurescriptsAllowedForOrgType(orgType: string): boolean {
```

Comment:
...then we can make `orgType` the actual type and not string.

Just having string here is bad b/c we can fat finger the value on the conditionals w/o getting any compiler error - then the function loses it's purpose.